src/controllers/WhatsappBot.ts

```diff
@@ -0,0 +1,54 @@
+import { google } from 'googleapis';
+import {config} from 'dotenv';
+import { Request, Response, NextFunction } from 'express';
+import twilio from 'twilio';
+
+config();
+
+// get the below from .env 
+const {
+  SID: accountSid, 
+  KEY: TwilloAuthToken,
+  APIKEY: googleApiKey,
+  CX: cx,
+} = process.env;
```

Comment:
There is a pattern of creating env, check the `.env.example` file and `config` folder